packages/bsky/src/api/app/bsky/feed/getActorLikes.ts

```diff
@@ -72,7 +72,7 @@ const skeleton = async (
     .innerJoin('like', 'like.subject', 'feed_item.uri')
     .where('like.creator', '=', actorDid)
 
-  const keyset = new FeedKeyset(ref('feed_item.sortAt'), ref('feed_item.cid'))
+  const keyset = new FeedKeyset(ref('like.sortAt'), ref('feed_item.cid'))
```

Comment:
this probably works, but keeping the secondary cursor on the same table can make pagination faster (it can stay in the index):

```suggestion
  const keyset = new FeedKeyset(ref('like.sortAt'), ref('like.cid'))
```